src/components/Contact.tsx

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import styled from "styled-components";
+import { useTranslation } from "react-i18next";
+import {
+  color,
+  layout,
+  space,
+  typography,
+  border,
+  flexbox,
+  background,
+  ColorProps,
+  LayoutProps,
+  SpaceProps,
+  TypographyProps,
+  BorderProps,
+  FlexboxProps,
+  BackgroundProps,
+} from "styled-system";
+
+const Main = styled.main<
+  ColorProps & SpaceProps & BackgroundProps & FlexboxProps & LayoutProps
+>`
+  min-height: 100vh;
+  overflow: hidden;
+  ${color};
+  ${space};
+  ${background};
+  ${flexbox};
+  ${layout};
+`;
+
+const Container = styled.div<
+  SpaceProps & BorderProps & LayoutProps & FlexboxProps & TypographyProps
+>`
+  ${space};
+  ${border};
+  ${layout};
+  ${flexbox};
+  ${typography}
+`;
+
+const H1 = styled.h1<ColorProps & SpaceProps & TypographyProps>`
+  text-transform: uppercase;
+  ${color};
+  ${space};
+  ${typography};
+`;
+
+const Email = styled.a<
+  ColorProps & SpaceProps & TypographyProps & BorderProps & LayoutProps
+>`
+  ${color};
+  ${space};
+  ${typography};
+  ${border};
+  ${layout};
+  &:hover {
+    background-color: #f4f4f6;
+    color: #000000;
+    transition-duration: 0.4s;
+  }
+`;
+
+const ContactUs = () => {
+  const textSizes = [1, 2, 3, 4];
+
+  const { t } = useTranslation();
+  const handleClick = () => {
+    var email = "xyz@abc.com";
+    window.location.href = "mailto:" + email;
+  };
+
+  return (
+    <Main
+      bg="black"
+      px={[1, 4, 6]}
+      pt={5}
+      display="flex"
+      alignItems="center"
+      justifyContent="center"
+    >
+      <Container
+        width={[200, 200, 300, 600]}
+        height={[200, 300, 400]}
+        borderWidth={1}
+        borderStyle="solid"
+        borderColor="athensGray"
+        p={5}
+      >
+        <H1 color="athensGray" fontSize={textSizes} pb={4}>
+          {t("contact.header")}
+        </H1>
+
+        <Email
```

Comment:
https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Creating_hyperlinks#E-mail_links

you don't need to do the "handleClick" thing if it's a mailto link.

Also might be good to open in new tab so the visitor doesn't get taken away from the site. To do this add `target="_blank"` attribute

Reply:
Yeah true! 

+ think I left the `handleClick` in there by mistake. I created it just in case we ended up going down the route of creating a form that triggers an email on submit. I was reading up about it and seems it may involve PHP? anyway, we're not doing a form so will park that for another day.